mkdocs.yml

```diff
@@ -165,6 +165,14 @@ nav:
         - 'Overview': 'integrations/databases/postgressql/overview.md'
       - 'Redis':
         - 'Overview': 'integrations/databases/redis/overview.md'
+    - 'Vector Databases':
+      - 'Overview': 'integrations/vector-databases/overview.md'
+      - 'Upstash':
+        - 'integrations/vector-databases/qdrant/overview.md'
+      - 'Qdrant':
+        - 'docs/integrations/vector-databases/qdrant/overview.md'
```

Comment:
The link here is wrong:

'docs/integrations/vector-databases/qdrant/overview.md'

It should be 

'integrations/vector-databases/qdrant/overview.md'